### ISSUE TITLE
Pronunciation of English words on screen

### ISSUE THREAD

**Issue by bnika** [grammatek/simaromur#125]
Do another round of adding missing English words and app names to the dictionary. For example "Talkback" is missing. Go systematically through the UI and transcribe missing words.

**Comment by bnika** [comment]
Did a round on this, re-open if we come across more missing words
